int/int_test.go

```diff
@@ -263,18 +402,18 @@ var _ = Describe("Integrationtests", func() {
 			})
 
 			It("changes the corresponding PrometheusRule", func() {
-				latestRouteMonitor, err := i.WaitForPrometheusRuleRef(expectedDependentResource, 20)
+				latestRouteMonitor, err := i.RouteMonitorWaitForPrometheusRuleRef(expectedDependentResource, 20)
 				Expect(err).NotTo(HaveOccurred())
 				latestRouteMonitor.Spec.Slo.TargetAvailabilityPercent = "99.5"
 				err = i.Client.Update(context.TODO(), &latestRouteMonitor)
 				Expect(err).NotTo(HaveOccurred())
-				_ , parsedSlo := latestRouteMonitor.Spec.Slo.IsValid()
-				err = i.WaitForPrometheusRuleCorrectSLO(expectedDependentResource, parsedSlo, 20)
+				_, parsedSlo := latestRouteMonitor.Spec.Slo.IsValid()
+				err = i.RouteMonitorWaitForPrometheusRuleCorrectSLO(expectedDependentResource, parsedSlo, 20, routeMonitor.Kind)
 				Expect(err).NotTo(HaveOccurred())
 			})
 		})
 
-		PWhen("the RouteMonitor Slo changes", func() {
+		When("the RouteMonitor Slo changes", func() {
```

Comment:
Great! this makes this test actually run!

sorry if this was an additional load on you to do